app/services/decisions/income_decision_tree.rb

```diff
@@ -1,4 +1,7 @@
 module Decisions
+  # TODO: remove this rubocop opt out when
```

Comment:
We discussed just chaining all the pages together not worrying about order. The comment I put on line 11 was intending to say this as page it directs the user to on line 12 isn't correct as per thee flow. 

I'm not worried either way.

Reply:
Can tidy up in future PR if its annoying - its all going to change anyway but I like the idea is gives us a "journey" to start working with.